Azure/sonic-buildimage
# Minigraph parser changes for storage backend acl

Signed-off-by: Neetha John <nejo@microsoft.com>



#### Why I did it
For storage backend, certain rules will be applied to the DATAACL table to allow only vlan tagged packets and drop untagged packets. 

#### How I did it
Create DATAACL table if the device is a storage backend device
To avoid ACL resource issues, remove EVERFLOW related tables if the device is a storage backend device

#### How to verify it
Added the following unit tests
 - verify that EVERFLOW acl tables is removed and DATAACL table is added for storage backend tor
 - verify that no DATAACL tables are created and EVERFLOW tables exist for storage backend leaf

#### Which release branch to backport (provide reason below if selected)
